zipkin-query-service/config/query-cassandra.scala

```diff
@@ -22,9 +22,16 @@ import org.twitter.zipkin.storage.cassandra.ZipkinRetryPolicy
 
 val contactPoints: Array[String] = sys.env.get("CASSANDRA_CONTACT_POINTS").getOrElse("localhost")
   .split(",")
+val cassandraAuth: String = sys.env.get("CASSANDRA_AUTH").getOrElse("")
```

Comment:
I wouldn't do the getOrElse things unless "cassandra/cassandra" is a builtin-user/password? Even then, I'd have people set it.

iotw use `sys.env.get("CASSANDRA_USER").isSet` or whatever to determine if the flags are set as we aren't using default values for this.